unit-tests/syncer/sw.py

```diff
@@ -15,6 +16,8 @@
 w = 640
 h = 480
 bpp = 2  # bytes
+#
+playback_status = None
 
 
 def init():
```

Comment:
No global gap_c,gap_d is defined, it's confusing how do you use it from the python test if it wasn't globally defined.

Reply:
Why not get all changeable parameters as arguments with default values, it will be better understandable, which parameters can be updated from outside this file.

Reply:
Added all the parameters set by init()/playback() as globals and added comment saying so.